test/primitives_test.dart

```diff
@@ -61,18 +71,44 @@ void main() {
           })) {
             break;
           }
-          await Future.delayed(const Duration(milliseconds: 10));
+          await Future.delayed(const Duration(milliseconds: 10), () {});
         }
         expect(await helperResult, equals('success'));
       });
     });
+
+    test('Timeout', () async {
+      final mutex = Mutex();
+
+      unawaited(
+          spawnLockedMutex(mutex.asSendable, const Duration(seconds: 10)));
+
+      /// give the isoalte a chance to start.
```

Comment:
- Typo: `isolate`.
- do not use doc comments inside functions.
- Please capitalize sentences in comments and end them with a `.`.

Reply:
This is still not resolved?